src/main/java/org/cbio/gdcpipeline/reader/MutationReader.java

```diff
@@ -78,13 +86,13 @@ public void open(ExecutionContext executionContext) throws ItemStreamException {
                 MutationRecord record = entry.getKey();
                 Set<String> caller = entry.getValue();
                 List<String> list = caller.stream().collect(Collectors.toList());
-                record.setCaller(StringUtils.join(list, '|'));
+                Map<String,String> addProp = new HashMap<>();
```

Comment:
It's better style to fully write out variable names - call it `additionalProperties`

Reply:
done